Cargo.toml

```
@@ -42,6 +42,7 @@ version = "0.2.2"
 
 [dev-dependencies]
 panic-halt = "0.2.0"
+cortex-m-semihosting = "0.3.1"
```

```suggestion
cortex-m-semihosting = "0.3.1"
```

Seems superfluous, no?

---

Apologise, it was from my previous semihosting based example